docs/reference/how-to/size-your-shards.asciidoc

```diff
@@ -72,12 +72,12 @@ space. While the exact amount resource usage of each additional field depends
 on the type of field, an additional 1024B in memory use  for each additional
```

Comment:
```suggestion
on the type of field, an additional 1024B in memory used for each additional
```

Should this be `used` instead of `use`?  Also I removed an additional space.

Reply:
Either is acceptable IMO ("memory use" works here as a nounal phrase)

Reply:
Let's remove that extra space then.